File: Api/Api/Core/Domain/Entities/TaskCard.cs
using System;
using System.Collections.Generic;

namespace Api.Core.Domain.Entities;

public partial class TaskCard
{
    public Guid Taskid { get; set; }

    public string? Name { get; set; }

    public string? Description { get; set; }

    public DateOnly? DueDate { get; set; }

    public string? Priority { get; set; }

    public Guid? Listid { get; set; }

    public virtual TaskList? List { get; set; }
}

File: Client/src/app/models/task-card.ts
export class TaskCard {
    TaskId: string;
    Name: string;
    Description: string;
    DueDate: Date;
    Priority: string;
    ListId: string;
  
    constructor(data: {
      TaskId: string,
      Name: string,
      Description: string,
      DueDate: Date,
      Priority: string,
      ListId: string
    }) {
      this.TaskId = data.TaskId;
      this.Name = data.Name;
      this.Description = data.Description;
      this.DueDate = data.DueDate;
      this.Priority = data.Priority;
      this.ListId = data.ListId;
    }
  }
  